packages/cloudflare/src/api/d1-tag-cache.ts

```diff
@@ -0,0 +1,151 @@
+import { debug, error } from "@opennextjs/aws/adapters/logger.js";
+import type { OpenNextConfig } from "@opennextjs/aws/types/open-next.js";
+import type { TagCache } from "@opennextjs/aws/types/overrides.js";
+
+import { getCloudflareContext } from "./cloudflare-context.js";
+
+// inlined during build
+const manifest = process.env.__OPENNEXT_CACHE_TAGS_MANIFEST;
```

Comment:
I don't think it's right to access `process.env` at top level as it will only be populated after the first request comes in.

Reply:
This is inlined during the build